package.json

```diff
@@ -2,7 +2,7 @@
   "name": "@datadog/native-iast-taint-tracking",
   "home": "https://github.com/DataDog/dd-native-iast-taint-tracking-js/blob/main/README.md",
   "repository": "git@github.com:DataDog/dd-native-iast-taint-tracking-js.git",
-  "version": "1.4.1",
+  "version": "1.5.0",
```

Comment:
Maybe it'd be correct to set the version to 1.4.2 since the modification was just header values. Anyway it's not really that important so I'm approving it and leaving the decision to make change to your judgement :)